getting-started.md

```diff
@@ -758,76 +758,73 @@ In this section, we will provide an overview of the supply chain security use ca
 
 ### Sign: Introducing Image Signing & Verification to your Supply Chain
 
-**Overview**
+#### Overview
+
+This component allows a platform operator to define a policy that will
+restrict unsigned images from running on clusters.
+To enforce the configured policies this component communicates with external
+container registries to verify signatures on container images and make a
+decision based on the results of this verification. In order to make admission
+decisions this component is implemented as a dynamic admission control webhook.
 
-This feature-set allows an application operator to define a policy that will restrict unsigned images from running on clusters.
-This is done using a dynamic admission control component on Kubernetes clusters.
-This component contains logic to communicate with external registries and verify signatures on container images,
-making a decision based on the results of this verification.
 Currently, this component supports cosign signatures and its key formats.
-It will work with open source cosign, kpack and Tanzu Build Service (which is what we will overview in this document).
+Although this component does not sign container images, you could use tools such
+as the cosign CLI, kpack, and Tanzu Build Service (which is what we will overview
```

Comment:
I am not sure if we are adding links to the tools, but it'd be great to add to the README.md of website for these tools.

Reply:
Oh I just saw we do have links to these tools down the doc ([lines 795-797](https://github.com/pivotal/docs-tap/pull/534/files/a9fa81cca80d572812a9cb1c7d54f5f59011a060#diff-f007f6cb65740b71d33ebd4511fa8727d48a472d135ffd1bbef978237f9e6fa7R795-R797)). Do you feel strongly about having them here too?

Reply:
Links are always helpful for users.